content/chapters/template-chapter/lecture/slides.mdpp

```diff
@@ -11,1121 +11,42 @@ revealOptions:
 
 ---
 
-## Processes
+## Slides hierarchy
```

Comment:
Ideally `slides.mdpp` should only contain the header above and `!INCLUDE` statements. All the actual content should be stored in separate files. So you should split this file into multiple files as well.

Reply:
What's the rationale for this?

Reply:
This would modularise our content, thus making it easier to review and reorder. Other trainers would also benefit from this because they will be able to use parts of our content easier if they want to.

Reply:
I expected the slides to be more monolithic, since they would be used for a single instance of the course.

Reply:
Markdown-PP takes care of that and merges them into a monolithic file. So I don't see a downside to storing them as multiple files.